packages/parcel-transformer/src/__tests__/transformer.parceltest.ts

```diff
@@ -68,6 +72,19 @@ it('transforms assets with babel plugin', async () => {
   `);
 }, 50000);
 
+it('transforms assets with custom resolver and statically evulates imports', async () => {
+  const parcel = getParcelInstance(customResolverFixtureRoot);
+  const { changedAssets } = await parcel.run();
+
+  const asset = Array.from(changedAssets.values()).find(
+    (asset) => asset.filePath === join(customResolverFixtureRoot, 'src/index.jsx')
+  );
+
+  const code = await asset?.getCode();
+
+  expect(code).toInclude('color:red');
```

Comment:
nit should we also check for font-size?

Reply:
um..good catch. I think I will remove 'font-size' from this test case. The test case of font-size has already been tested at https://github.com/atlassian-labs/compiled/blob/master/packages/parcel-transformer/src/__tests__/transformer.parceltest.ts#L46. The purpose of this test is to ensure we are able to statically evaluate color from another file.